projects/storefrontlib/src/cms-components/organization/users/user-approvers/user-approvers.component.html

```diff
@@ -3,21 +3,21 @@
   [links]="[
     {
       cxRoute: 'userDetails',
-      name: 'user.details',
+      name: 'userForm.details',
```

Comment:
that's not form component

Reply:
The name property is used by fake tabs component to translations only, and one could argue that tab elements are form elements. User translations should only contain user properties.